platform/ui/src/assets/icons/icon-nav-studies.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="25" height="25" viewBox="0 0 25 25">
```

Comment:
@rodrigolabs & @galelis, having trouble remembering. Do we want width/height cleared from icons? The only thing that's important is `viewBox`, right? Having these removed would make it easier to scale w/ styles

Reply:
Having trouble remembering if these matter for aspect ratio, or if you try to fix that by using a grid system when creating the SVG